packages/taquito-local-forging/src/taquito-local-forging.ts

```diff
@@ -22,7 +22,7 @@ export * from './interface';
 export { VERSION } from './version';
 export { ProtocolsHash } from './protocols';
 
-const PROTOCOL_CURRENT = ProtocolsHash.PtMumbai2;
+const PROTOCOL_CURRENT = ProtocolsHash.Proxford;
```

Comment:
correct me if I'm wrong, but isn't the shortened protocol hash `ProxfordS` instead?

Reply:
it's defined [here](https://github.com/ecadlabs/taquito/blob/oxford/packages/taquito-local-forging/src/protocols.ts#L18) 
<img width="647" alt="image" src="https://github.com/ecadlabs/taquito/assets/106410553/5cd16603-a7b3-4c47-a033-66f3aeda5212">

Reply:
Counted some 6 some 7. But you're right the recent ones are all 7 char, will update